doc/sphinx/refman-preamble.rst

```diff
@@ -11,60 +11,18 @@
 
 .. role:: smallcaps
 
-.. |A_1| replace:: `A`\ :math:`_{1}`
-.. |A_n| replace:: `A`\ :math:`_{n}`
-.. |arg_1| replace:: `arg`\ :math:`_{1}`
-.. |arg_n| replace:: `arg`\ :math:`_{n}`
-.. |bdi| replace:: :math:`\beta\delta\iota`
-.. |binder_1| replace:: `binder`\ :math:`_{1}`
-.. |binder_n| replace:: `binder`\ :math:`_{n}`
-.. |binders_1| replace:: `binders`\ :math:`_{1}`
-.. |binders_n| replace:: `binders`\ :math:`_{n}`
-.. |C_1| replace:: `C`\ :math:`_{1}`
 .. |c_1| replace:: `c`\ :math:`_{1}`
-.. |C_2| replace:: `C`\ :math:`_{2}`
 .. |c_i| replace:: `c`\ :math:`_{i}`
 .. |c_n| replace:: `c`\ :math:`_{n}`
-.. |Cic| replace:: :smallcaps:`Cic`
-.. |class_1| replace:: `class`\ :math:`_{1}`
-.. |class_2| replace:: `class`\ :math:`_{2}`
+.. |Cic| replace:: CIC
 .. |Coq| replace:: :smallcaps:`Coq`
```

Comment:
Why change the replace for `|Cic|` but not eg this one?

Reply:
For some reason, the smallcaps do not render as expected in the HTML version of the manual. This leads CIC to be written Cic which is really wrong (this is an acronym). The fact that Coq is displayed as Coq in the HTML version and with small caps in the PDF version does not seem to be a problem to me.

Reply:
Maybe it should be `` .. |Cic| replace:: :smallcaps:`CIC` `` then

Reply:
But ``` :smallcaps:`CIC` ``` is equivalent to `CIC`, isn't it? IINM the point of small caps is to print *non-capitalized* letters as small capital letters.

Reply:
Looks like the stylesheet is missing a rule:

```css
.smallcaps {
    font-variant: small-caps;
}
```

That seems to do the trick for me.  (But I think CIC should be in all caps anyway, not small caps).

Reply:
Do you want to add this to this PR or should I merge it as is?

Reply:
I'll add it to this PR.

Reply:
BTW, can you remind me why we have two `coqdoc.css` (one in `doc/sphinx/` and one in `doc/sphinx/_static/`)? The one that needs to be changed in that case is the latter, but the former is easier to find and thus confusing.

Reply:
Actually, this will be part of another PR because as soon as we correctly render smallcaps, we start to notice inconsistencies in the text, including in the intro.

Reply:
> BTW, can you remind me why we have two coqdoc.css (one in doc/sphinx/ and one in doc/sphinx/_static/)? 

No idea.  The first one wasn't part of my coq-rst branch AFAICT, and I'm not sure why it was added.